lib/js/icons/fontawesome.ts

```diff
@@ -338,6 +339,7 @@ export const FONTAWESOME_ICONS = {
 	FA_SHARE: faShare,
 	FA_SHARE_NODES: faShareNodes,
 	FA_SHIELD_HALVED: faShieldHalved,
+	FA_SIDEBAR_FLIP: faSidebarFlip,
```

Comment:
do we have to use flipped version? We could flip the icon in CSS or via props on Icon. This way, if we find a use-case for regular `SIDEBAR` icon, we won't need to import 2 separate assets